mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go

```diff
@@ -287,6 +314,10 @@ func resourceLoggingBucketConfigRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error setting cmek_settings: %s", err)
 	}
 
+	if err := d.Set("index_configs", flattenIndexConfigs(indexConfigs.([]interface{}))); err != nil {
```

Comment:
```suggestion
	if err := d.Set("index_configs", flattenIndexConfigs(res["indexConfigs"].([]interface{}))); err != nil {
```

Reply:
(I haven't run these at all- you may want to make the changes locally and ensure it builds / a test passes to potentially save a round trip!)

Reply:
Yeah, have been for all builds. Should've paid close attn to latest one as it was the cache being on that tripped this over. Should've looked close to see the test results were cached for above commit

Reply:
Oh, you're all good! I've just had a handful of cases like this turn into multiple day-long round trips of minor syntax changes enough times that I try to warn that I am suggesting a literal fix to a single syntax problem and not guaranteeing the code actually compiles or anything.